src/Parser.php

```diff
@@ -116,6 +122,21 @@ public function parse(TokenStream $stream, $test = null, bool $dropNeedle = fals
         return $node;
     }
 
+    public function shouldIgnoreUnknownTwigCallables(): bool
+    {
+        return $this->ignoreUnknownTwigCallables;
+    }
+
+    public function subparseIgnoreUnknownTwigCallables($test, bool $dropNeedle = false): void
+    {
+        $this->ignoreUnknownTwigCallables = true;
+        try {
+            $this->subparse($test, $dropNeedle);
+        } finally {
+            $this->ignoreUnknownTwigCallables = false;
```

Comment:
this should reset the previous state of the property instead, in case such tags are nested

Reply:
I've added some tests, do you have any example in mind that would fail?

Reply:
Right, fixed.